content/02.introduction.md

```diff
@@ -1,2 +1,45 @@
 ## Introduction {.page_break_before} 
-This is where the introduction will be.
+### General Background
+On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus disease 2019 (COVID-19) [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4].
```

Comment:
The citation keys don't support the `=` character.  You could use a citation tag or simply remove `?sfvrsn=20a99c10_4` from this citation and others like it.

Reply:
Thank you so much! I'll fix that!